addons/website_slides/controllers/main.py

```diff
@@ -841,6 +842,10 @@ def slide_preview(self, slide_id):
 
     @http.route(['/slides/slide/send_share_email'], type='json', auth='user', website=True)
     def slide_send_share_email(self, slide_id, email, fullscreen=False):
+        email_ids = email.split(",")
```

Comment:
(Picky remark: if a file use single quote, use single quote for consistency)
```suggestion
        email_ids = email.split(',')
```